meetings/approvals/tests.py

```diff
@@ -1,12 +0,0 @@
-from django.test import TestCase
-from unittest import skip
-
-
-class TestApprovals(TestCase):
```

Comment:
This is a place for future tests, please do not remove.

 @Tav0 It isn't referenced in the `urls.py` because it is never meant to be seen as a URL. It runs with `inv test_all` and won't be skipped once #67 is finished. The idea is that I outlined tests so that they could be filled in, not deleted.